packages/_site/src/pages/components/popover/index.tsx

```diff
@@ -0,0 +1,390 @@
+				description="A popover is a button that displays content in one of four directions; right, left, top, bottom."
+			/>
+
+			<div {...stylex.props(styles.threeColumnLayout)}>
+				<ContentsSidebar ref={divRef} />
+
+				<div {...stylex.props(styles.middleColumn)}>
+					<H2 id="installation">Installation</H2>
+					<Divider />
+					<div>
+						<Tabs defaultValue="tab-1">
+							<TabsList>
+								<TabsTrigger value="tab-1">Automatic</TabsTrigger>
+								<TabsTrigger value="tab-2">Manual</TabsTrigger>
+							</TabsList>
+							<TabsContent value="tab-1">
+								<div {...stylex.props(styles.blockWrapper)}>
+									<ComponentInfo
+										install="pnpm add @controlkit/popover"
+										npmjs="https://www.npmjs.com/package/@controlkit/popover"
+										npmTitle="@controlkit/popover"
+										source="https://github.com/clearfeld/control-ui/tree/main/packages/popover"
+									/>
+								</div>
+							</TabsContent>
+							<TabsContent value="tab-2">
+								<div {...stylex.props(styles.stepBlock)}>
+									<H5>1. Copy and paste the following code into your project.</H5>
+
+									<CodeBlock
+										language="tsx"
+										url={"https://raw.githubusercontent.com/clearfeld/control-ui/main/packages/popover/lib/index.tsx"}
+									/>
+								</div>
+							</TabsContent>
+						</Tabs>
+					</div>
+
+					<H2 id="examples">Examples</H2>
+					<Divider />
+
+					<div>
+						<ExampleBlock
+							title="Top"
+							description="Button that displays content to the top."
+						/>
+						<div {...stylex.props(styles.codeWrapper)}>
+							<PreviewBlock>
+								<Popover>
+									<PopoverTrigger
+										style={{
+											width: "20rem",
+										}}
+										asChild
+									>
+										<Button>Open Top</Button>
+									</PopoverTrigger>
+
+									<PopoverContent
+										style={{
+											width: "20rem",
+										}}
+										side={"top"}
+									>
+										<div
+											style={{
+												border: "0.0625rem solid var(--sidebar-color-bg-hover)",
+												backgroundColor: "var(--color-bg-compliment)",
+												height: "20rem",
+												textAlign: "center",
+												borderRadius: "0.25rem",
+											}}
+										>
+											Place content for the popover here.
+										</div>
+									</PopoverContent>
+								</Popover>
+							</PreviewBlock>
+
+							<CodeBlock
+								language="tsx"
+								code={`import { Popover, PopoverContent, PopoverTrigger } from "@controlkit/popover";
+import { Button } from "@controlkit/button";
+
+export default function CardGroup() {
+	return (
+		<Popover>
+			<PopoverTrigger
+				style={{
+					width: "20rem",
+				}}
+				asChild
+			>
+				<Button>Open Top</Button>
+			</PopoverTrigger>
+
+			<PopoverContent
+				style={{
+					width: "20rem",
+				}}
+				side={"top"}
+			>
+				<div
+					style={{
+						border: "0.0625rem solid var(--sidebar-color-bg-hover)",
+						backgroundColor: "var(--color-bg-compliment)",
```

Comment:
Examples shouldn't have / use our internal site specific css variables. They should be general enough that they can be copied into any project without additional setup post copy.